art/estimators/mxnet.py

```diff
@@ -81,3 +81,18 @@ def loss(self, x: np.ndarray, y: np.ndarray, **kwargs) -> np.ndarray:
         :rtype: Format as expected by the `model`
         """
         raise NotImplementedError
+
+    def clone_for_refitting(self) -> 'MXEstimator':  # lgtm [py/inheritance/incorrect-overridden-signature]
+        """
+        Create a copy of the estimator that can be refit from scratch.
+
+        :return: new estimator
+        """
+        raise NotImplementedError
+
+    def reset(self) -> None:
+        """
+        Resets the weights of the estimator so that it can be refit from scratch.
+
+        """
+        raise NotImplementedError
```

Comment:
I think we should not add these lines as they remove the requirement from classes inheriting from this class of implementing these new methods.